Automattic/woocommerce-payments
# Fix: Cart contents get cleared after loggin in

Fixes #5491 

#### Changes proposed in this Pull Request
- Updated jetpack-related dependencies to fix the cart clear issue.
- See comment https://github.com/Automattic/woocommerce-payments/issues/5491#issuecomment-1784811631








#### Testing instructions





- Enable account creation in `woocommerce > Setting > Accounts and Privacy`
- Create an account in shop > My Account
- Open an incognito window, add items to cart
- Login with newly created user
- Check if the cart content remains.

-------------------


**Post merge**
